Fedya/FANUC Client/RobotRecieveDifinition.inl

```diff
@@ -28,20 +28,20 @@ int RobotRecieve<T>::readCoord()
 	}
 	else
 	{
-		_sbuf += recvbuf;
+		_stringBuffer += recvbuf;
 		int coordsNumber = 0;
-		for (int i = 0; i < _sbuf.size(); i++)
+		for (int i = 0; i < _stringBuffer.size(); i++)
```

Comment:
**(must be change)**: change this function to Vasily's parsing form utility lib.